Scientific_workflows/WIT_Metrics/wit_metrics.ipynb

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
<p>To make this notebook easier to understand, could you please add a description of what the WIT dataframe looks like? This will help people build off your work here in future.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/772/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
We have a document somewhere to describe all the things. I shall chuck in the link of the doc.